server/meshmodel/kubernetes/relationships/hierarchical_inventory.json

```diff
@@ -1077,6 +1077,40 @@
           "to": []
         }
       }
+    },
+    {
+      "allow": {
+        "from": [
+          {
+            "kind": "IngressClass",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatorRef": [
+                ["name"]
+              ],
+              "description": "IngressClass establishes an inventory relationship with Ingress. IngressClass is a resource to provide default class for Ingress. When a IngressClass is dropped on Ingress specification of Ingress will be updated and configure the class of the Ingress controller."
```

Comment:
Please don't use "dropped". Such phrasing is only applicable to a visual designer interface.